nevergrad/functions/__init__.py

```diff
@@ -5,7 +5,6 @@
 
 from .functionlib import ArtificialFunction as ArtificialFunction
 from .functionlib import FarOptimumFunction as FarOptimumFunction
-from .functionlib import FarOptimumFunction as PBT
```

Comment:
@teytaud this means that PBT has actually never been used in experiments before

Reply:
we have results so it has been run at some point, but maybe never with master.